src/SAREhub/Client/Processor/LogProcessor.php

```diff
@@ -55,8 +55,18 @@ public function getLogLevel(): string
 
     public function setLogLevel(string $logLevel)
     {
+        $this->isValidLogLevel($logLevel);
         $this->logLevel = $logLevel;
-        return $this;
+    }
+
+    private function isValidLogLevel(string $logLevel): void
```

Comment:
too complicated 

if (!in_array(strtolower($logLevel), $logLevelReflection->getConstants())) {
  // throw exception
}

Reply:
add test for logLevel valid value but is like "Info" or "iNFO"